stl/inc/mdspan

```diff
@@ -242,10 +240,10 @@ extents(_Integrals... _Ext)
     -> extents<size_t, conditional_t<true, integral_constant<size_t, dynamic_extent>, _Integrals>::value...>;
 
 template <class _Type>
-constexpr bool _Is_extents = false;
+constexpr bool _Is_extents_v = false;

```

Comment:
```suggestion
inline constexpr bool _Is_extents_v = false;
```
I think `inline` can be used unconditionally, instead of `_INLINE_VAR`, since this is C++23. Occurs below.